### ISSUE TITLE
GeoTags on the home page

### ISSUE THREAD

**Issue by kolatubosun** [Yorubaname/yorubaname-website#60]
Let's make it such that one can click on a certain tag, e.g. "Ijebu" or "Foreign-Arabic" or "General" (which are all geotags already placed on each name entry) and have the dictionary list only those names. That way, if the user is looking for only Ijesha names, they can just click on the "Ìjẹ̀ṣà" tag and have the names before them.

**Comment by Flipponachi** [comment]
Am sorry but I couldn't find on the website where such feature is listed, as I want to contribute to this

**Comment by kolatubosun** [comment]
Hi @Flipponachi, each name entry has a geolocation tag. My point here is that we could have a front-facing way of using these tags to search for names.

**Comment by Flipponachi** [comment]
Sorry its took quite a while to respond @kolatubosun, so in essence we are creating another method to search names, as in users can search names that are common to specific areas based on the geolocation? First off, how many tags are there? Secondly aren't the tags supposed to link back to yoruba regions ? I think thats all for now

**Comment by kolatubosun** [comment]
Yes @Flipponachi It's another method to search for names. There are about seven or eight tags in all. You can find them if you try to add a name and you go to the "Geolocation" part.
